tests/attr/neuron/test_neuron_deeplift_basic.py

```diff
@@ -114,8 +118,22 @@ def test_relu_deepliftshap_with_custom_attr_func(self):
         expected = (torch.zeros(3, 3), torch.zeros(3, 3))
         self._relu_custom_attr_func_assert(neuron_dl, inputs, baselines, expected)
 
-    def _relu_custom_attr_func_assert(self, attr_method, inputs, baselines, expected):
-        def custom_attr_func(multipliers, inputs, baselines):
+    def _relu_custom_attr_func_assert(
+        self,
+        attr_method: Any,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
+        ],
+        expected,
+    ) -> None:
+        def custom_attr_func(
+            multipliers: TensorOrTupleOfTensors,
+            inputs: TensorOrTupleOfTensors,
+            baselines: Optional[
+                Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
+            ] = None,
+        ) -> TensorOrTupleOfTensors:
```

Comment:
This return type should be Tuple[Tensor, ...]

Reply:
Thanks!